apps/academy/src/styles.css

```diff
@@ -1,5 +1,27 @@
 @import "ui/styles.css";
 
+/* Apply the background to the entire page */
+
+html {
+  margin: 0;
+  padding: 0;
+  width: 100vw;
+  overflow-x: hidden; /* Prevent horizontal scroll */
+}
+
+body {
+  margin: 0; /* Remove default margin */
+  padding: 0;
+  min-height: 100vh;
+  width: 100%;
+  background-image: url("/bg_home.png");
```

Comment:
@rohann06 the background image is appearing on every track and on every lesson page:

![image](https://github.com/user-attachments/assets/8ea90a04-69af-4897-b790-995887edbfd7)

can you handle it to behave as figma design?

![image](https://github.com/user-attachments/assets/9fa64844-38c4-432c-9f62-b8dfbb6b66b3)

With black background for track page and for lessons page.